src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -160,6 +160,27 @@ function getAjaxStudentList(ajaxPreloadImage) {
     });
 }
 
+/**
+ * Gets enroll status through an AJAX request.
+ * @returns {Promise} the state of the result from the AJAX request
+ */
+function getAjaxEnrollStatus() {
```

Comment:
I thought the purpose of this PR compared to the previous one is that the AJAX endpoint will acutally enroll students and report errors if there are any? If so, shouldn't the function be called just `enrollStudents` or something similar instead of `get...Status`?